shell/platform/android/android_context_gl.cc

```diff
@@ -206,12 +206,19 @@ std::unique_ptr<AndroidEGLSurface> AndroidContextGL::CreateOnscreenSurface(
     fml::RefPtr<AndroidNativeWindow> window) const {
   EGLDisplay display = environment_->Display();
 
-  const EGLint attribs[] = {EGL_NONE};
-
-  EGLSurface surface = eglCreateWindowSurface(
-      display, config_, reinterpret_cast<EGLNativeWindowType>(window->handle()),
-      attribs);
-  return std::make_unique<AndroidEGLSurface>(surface, display, context_);
+  if (window->IsOffscreen()) {
```

Comment:
This is imprecise - an `AndroidNativeWindow` can be backed by a real window surface that is positioned offscreen.

Change this to something like `IsFakeWindow` and emphasize that this is only provided to support tests and will not happen on a real device.

Reply:
Done, the docstring mentions it's for tests.